.gitignore

```diff
@@ -109,3 +109,5 @@ dist/
 out-tsc/
 
 .DS_Store
+
```

Comment:
This PR looks awesome. Love that you were able to get the dynamic form stuff started and working!

Reply:
Thanks Ben!